src/logging.cpp

```diff
@@ -130,8 +130,7 @@ struct CLogCategoryDesc
     std::string category;
 };
 
-const CLogCategoryDesc LogCategories[] =
-{
+const std::array<CLogCategoryDesc, 28> LogCategories{{
```

Comment:
Is there a reason to change this? The commit doesn't provide a motivation, and it is actually dangerous because it opens the door for silent merge conflicts. Someone removing an element will *not* result in a compile failure. See also this diff, which compiles:

```diff
diff --git a/src/logging.cpp b/src/logging.cpp
index c604444acc..b193b77da3 100644
--- a/src/logging.cpp
+++ b/src/logging.cpp
@@ -130,7 +130,7 @@ struct CLogCategoryDesc
     std::string category;
 };
 
-const std::array<CLogCategoryDesc, 28> LogCategories{{
+const std::array<CLogCategoryDesc, 29> LogCategories{{
     {BCLog::NONE, "0"},
     {BCLog::NONE, "none"},
     {BCLog::NET, "net"},

Reply:
Good point. Dropped this change.